app/src/main.cpp

```diff
@@ -343,15 +319,12 @@ int main(int argc, char* argv[])
     // Force dot separator on numbers because some localizations
     // uses comma as separator.
     std::setlocale(LC_NUMERIC, "en_US.UTF-8");
+
+    PlatformHandler::initialise();
+
```

Comment:
I’d prefer having this after the resource initialisation for core_lib, since technically PlatformHandler is from core_lib and might use its resources (even though it currently does not). That said, it’s a cosmetic thing so I won’t block the PR over it.

Reply:
Good point. I just put it above because I wasn't sure if the gstreamers stuff needed to be set up before core_lib. As long as it's not an issue, I agree it's best to be after.